benchmarks/datasets/join/generate.sql

```diff
@@ -0,0 +1,118 @@
+\set rows :rows
+\echo 'Generating' :rows 'rows in a facts table, and' :rows '/ 8 rows in two dimension tables.'
+
+
+-- Create tables
+
+DROP TABLE IF EXISTS pages CASCADE;
+CREATE TABLE pages (
+    "id" TEXT PRIMARY KEY,
+    "fileId" TEXT,
+    "content" TEXT,
+    "title" TEXT,
+    "parents" TEXT,
+    "sizeInBytes" BIGINT,
+    "createdAt" TIMESTAMP
+);
+
+DROP TABLE IF EXISTS files CASCADE;
+CREATE TABLE files (
+    "id" TEXT PRIMARY KEY,
+    "documentId" TEXT,
+    "content" TEXT,
+    "title" TEXT,
+    "parents" TEXT,
+    "sizeInBytes" BIGINT,
+    "createdAt" TIMESTAMP
+);
+
+DROP TABLE IF EXISTS documents CASCADE;
+CREATE TABLE documents (
+    "id" TEXT PRIMARY KEY,
+    "parents" TEXT,
+    "content" TEXT,
+    "title" TEXT,
+    "createdAt" TIMESTAMP
+);
```

Comment:
As far as I remember, The schema is supposed to be a hierarchy of documents -> files -> pages, and the content is only in `pages`. Also, `files` and `pages` don't have parents. But, if the workload is representative of customer's performance, I think it's fine.

Reply:
Yea, this is representative... a bit odd, but all of the tables are wide, and all of them have a title and content.